phpseclib/Math/PrimeField.php

```diff
@@ -41,6 +41,11 @@ class PrimeField extends FiniteField
      */
     protected $instanceID;
 
+    /**
+     * @var BigInteger
+     */
+    protected $modulo;
```

Comment:
I think we can just delete `$this->modulo = $modulo;` from the constructor. Doesn't appear to be used anywhere else.